Tensile/Source/lib/include/Tensile/ContractionSolution.hpp

```diff
@@ -34,6 +34,10 @@
 #include <Tensile/DataTypes.hpp>
 #include <Tensile/Predicates.hpp>
 
+#include <boost/program_options.hpp>
```

Comment:
Boost cannot be used inside the lib directory as that imposes boost as a dependency on all users.